src/python/platforms/android/adb.py

```diff
@@ -370,6 +370,12 @@ def start_cuttlefish_device():
   launch_cvd_command_line = (
       f'{launch_cvd_path} -daemon -memory_mb {device_memory_mb} '
       '-report_anonymous_usage_stats Y')
+  if use_kernel:
+    kernel_path = os.path.join(cvd_dir, 'bzImage')
+    initramfs_path = os.path.join(cvd_dir, 'initramfs.img')
+    launch_cvd_command_line += (
+        f' -kernel_path={kernel_path} -initramfs_path={initramfs_path}')
```

Comment:
where's the default one used if these aren't set?

Reply:
There is a default one built in Cuttlefish boot img, we need to override it with KASAN build to make kernel fuzzing work.